persistence-jdbc/core/src/test/scala/com/lightbend/lagom/internal/persistence/jdbc/SlickDbTestProvider.scala

```diff
@@ -23,11 +25,11 @@ object SlickDbTestProvider {
   }
 
   /** Builds Slick Database (with AsyncExecutor) and bind it as JNDI resource for test purposes  */
-  def buildAndBindSlickDb(baseName: String, lifecycle: ApplicationLifecycle): Unit = {
+  def buildAndBindSlickDb(baseName: String, lifecycle: ApplicationLifecycle, coordinatedShutdown: CoordinatedShutdown)(implicit executionContext: ExecutionContext): Unit = {
     val dbName = s"${baseName}_${Random.alphanumeric.take(8).mkString}"
     val db     = Databases.inMemory(dbName, config = Map("jndiName" -> JNDIName))
     lifecycle.addStopHook(() => Future.successful(db.shutdown()))
```

Comment:
do we still need this?

Reply:
@ignasi35, if we still need it, just remove the `block-merge`. It's approved.

Reply:
Hmmm, this one was never required because the following statement internally registers a shutdown hook/task over the `db`. I've rewritten it into a Coordinated Shutdown task but we might as well remove that.

It doesn't harm as long as it runs in the correct phase.

Reply:
I've reconsidered and removed it.